src/java.desktop/share/classes/javax/swing/JFileChooser.java

```diff
@@ -2078,4 +2078,20 @@ public void hierarchyChanged(HierarchyEvent e) {
             }
         }
     }
+
+    @Override
```

Comment:
No javadoc .. are you hoping to get away without a CSR ?
I think a CSR would be a good idea anyway because after 25 years of Swing ignoring this
on a JFileChooser, you are going to support it. And if an app had set it and not realised then
it'll see a change. 
So I'm not going to suggest javadoc but I am going to suggest a PR for compatibility concerns.
Then there's always the option to back it out if there's an issue.

Reply:
Sure, will raise a CSR for that.